docs/references/_attachments/ic.did

```diff
@@ -323,10 +323,17 @@ type schnorr_public_key_result = record {
     chain_code : blob;
 };
 
+type schnorr_aux = variant {
+    bip341: record {
+      merkle_root_hash: blob;
+   }
+};
+
 type sign_with_schnorr_args = record {
     message : blob;
     derivation_path : vec blob;
     key_id : record { algorithm : schnorr_algorithm; name : text };
+    aux: opt schnorr_aux;
```

Comment:
Maybe write it out as auxiliary_parameters so that it's a bit clearer (for people that do not spend their free time reading CS theory papers)?